onecodex/lib/enums.py

```diff
@@ -0,0 +1,13 @@
+ABUNDANCE_FIELDS = {"abundance", "abundance_w_children"}
+
+ALPHA_DIVERSITY_METRICS = {"simpson", "chao1", "shannon"}
+
+BETA_DIVERSITY_METRICS = {
+    "jaccard",
+    "braycurtis",
+    "cityblock",
+    "weighted_unifrac",
+    "unweighted_unifrac",
+}
+
+FIELDS = {"auto", "abundance", "readcount", "readcount_w_children"}
```

Comment:
Can we make all of those enums? We compare to a string everywhere in the codebase.